src/i18n/sv_SE.ts

```diff
@@ -808,7 +808,7 @@ export default {
     alliance: 'Allians',
     premiumNotice:
         'Den här funktionen utökar en premiumfunktion i spelet och är därför endast tillgänglig för spelare med ett Larmcentralen-spelet premiumkonto!',
-    credits: 'Krediter',
+    credits: 'credits',
```

Comment:
@Suchty112 Why did you change this translation? That one is correct when it's in sv_SE

Reply:
Whoops. fixing!

Reply:
"credits: 'Krediter'," that is, of course, correct. 
Sorry, my programm has renamed too much